Allow two more domctls to IS_PRIV_FOR guests, that stubdoms apparently need.

diff -r 62fa1bc6c99a xen/common/domctl.c
--- a/xen/common/domctl.c	Thu Jul 08 10:28:14 2010 +0100
+++ b/xen/common/domctl.c	Thu Jul 08 10:35:18 2010 +0100
@@ -231,6 +231,8 @@
 
     switch ( op->cmd )
     {
+    case XEN_DOMCTL_getdomaininfo:
+    case XEN_DOMCTL_settimeoffset:
     case XEN_DOMCTL_ioport_mapping:
     case XEN_DOMCTL_memory_mapping:
     case XEN_DOMCTL_bind_pt_irq:
